packages/safe-apps-wagmi/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # @gnosis.pm/safe-apps-wagmi
 
+## Unreleased - 1.3.0
```

Comment:
Could you please remove this and generate a changeset instead? The changeset then will be used to generate the changelog. For more instructions you can check the changeset bot comment

Reply:
The bot will also take care of all the versioning so please revert all versioning changes

Reply:
> I have read the CLA Document and I hereby sign the CLA

recheckcla

Reply:
done

Reply:
Could you please add a changeset? https://github.com/safe-global/safe-apps-sdk/pull/441#issuecomment-1401807844

Reply:
@mikhailxyz done, does this look correct?